lib/simplifyExpression/index.js

```diff
@@ -1,7 +1,9 @@
 const math = require('mathjs');
 const stepThrough = require('./stepThrough');
+const Substitute = require('../Substitute');
 
-function simplifyExpressionString(expressionString, debug=false) {
+function simplifyExpressionString(expressionString, debug=false, scope={}) {
```

Comment:
also I think we should replace `scope` with `options` - I'll say more about this in a top level comment on this PR